src/__tests__/sources.int.test.ts

```diff
@@ -550,4 +555,81 @@ describe('/sources', () => {
 			});
 		});
 	});
+	describe('DELETE /:sourceId', () => {
+		it('requires authentication', async () => {
+			await agent.delete('/sources/:sourceId').expect(401);
+		});
+
+		it('requires administrator role', async () => {
+			await agent.delete('/sources/:sourceId').expect(401);
+		});
+
+		it('deletes exactly one source that has no proposals associated with it', async () => {
+			const changemaker = await createChangemaker(db, null, {
+				taxId: '11-1111111',
+				name: 'Example Inc.',
+				keycloakOrganizationId: null,
+			});
+			const localSource = await createSource(db, null, {
+				changemakerId: changemaker.id,
+				label: 'Example Inc.',
+			});
+			const before = await loadTableMetrics('sources');
+
+			await agent
+				.delete(`/sources/${localSource.id}`)
+				.type('application/json')
+				.set(adminUserAuthHeader)
+				.expect(200);
+
+			const after = await loadTableMetrics('sources');
+
+			expect(before.count).toEqual(2);
+			expect(after.count).toEqual(1);
+		});
+
+		it('throws an error when it tries to delete a source that is associated with a proposal', async () => {
```

Comment:
Let's instead say `Returns 422 Unprocessable Content`

This is more descriptive and more accurate (it doesn't throw an error!)